modules/nw-installing-external-dns-operator-cli.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-installing-external-dns-operator-cli_{context}"]
+= Installing the External DNS Operator by using the CLI
+
+You can install the External DNS Operator by using the CLI.
+
+.Prerequisites
+
+* You are logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* You are logged into the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create an `OperatorGroup` object:
+
+.. Create a YAML file that defines the `OperatorGroup` object:
++
+.Example `operatorgroup.yaml` file
```

Comment:
There is one step missing - the creation of the `external-dns-operator` namespace. It should be the first step. You can see how it's done in AWS LB Operator module.